content/en/notification-manager/notification-manager.md

```diff
@@ -0,0 +1,404 @@
+
+Notification Manager now supports three type receivers, email, Wechat Work, slack. Only administrator can config receivers.
+
+#### Email
+
+If a tenant named test-user want to receive notification with email, it must be create a email receiver like this.
+
+```
+cat <<EOF | kubectl apply -f -
+apiVersion: notification.kubesphere.io/v1alpha1
+kind: EmailConfig
+metadata:
+  labels:
+    app: notification-manager
+    type: tenant
+    user: test-user
+  name: test-user-config
+  namespace: kubesphere-monitoring-system
+spec:
+  authPassword:
+    key: password
+    name: test-user-secret
+    namespace: kubesphere-monitoring-system
+  authUsername: abc1
+  from: abc1@xyz.com
+  requireTLS: true
+  smartHost:
+    host: imap.xyz.com
+    port: "25"
+
+---
+apiVersion: notification.kubesphere.io/v1alpha1
+kind: EmailReceiver
+metadata:
+  labels:
+    app: notification-manager
+    type: tenant
+    user: test-user
+  name: test-user-receiver
+  namespace: kubesphere-monitoring-system
+spec:
+  emailConfigSelector:
+    matchLabels:
+      type: tenant
+      user: test-user
+  to:
+  - abc2@xyz.com
+  - abc3@xyz.com
+
+---
+apiVersion: v1
+data:
+  password: dGVzdA==
+kind: Secret
+metadata:
+  labels:
+    app: notification-manager
+  name: test-user-secret
+  namespace: kubesphere-monitoring-system
+type: Opaque
+EOF
+```
+
+The emailConfigSelector is a selector to selecte EmailConfig for email receiver, if the emailConfigSelector is nil, receiver will use default email config. You can create a default email config like this.
+
+```
+cat <<EOF | kubectl apply -f -
+apiVersion: notification.kubesphere.io/v1alpha1
+kind: EmailConfig
+metadata:
+  labels:
+    app: notification-manager
+    type: default
+  name: default-email-config
+  namespace: kubesphere-monitoring-system
+spec:
+  authPassword:
+    key: password
+    name: default-email-secret
+    namespace: kubesphere-monitoring-system
+  authUsername: default
+  from: default@xyz.com
+  requireTLS: true
+  smartHost:
+    host: imap.xyz.com
+    port: "25"
+
+---
+apiVersion: v1
+data:
+  password: dGVzdA==
+kind: Secret
+metadata:
+  labels:
+    app: notification-manager
+  name: default-email-secret
+  namespace: kubesphere-monitoring-system
+type: Opaque
+```
+
+Tanten email receiver only receive notification happend in the namespace which tanten in. If you want to receive notification without namespace, you can create a global email receiver like this.
+
+```
+apiVersion: notification.kubesphere.io/v1alpha1
+kind: EmailReceiver
+metadata:
+  labels:
+    app: notification-manager
+    type: global
+  name: global-email-receiver
+  namespace: kubesphere-monitoring-system
+spec:
+  to:
+  - global@xyz.com
+EOF
+```
+
+>Global email receiver use default email config.
+
+#### Wechat Work
+
+Notification Manager supports sending notification to Wechat Work. 
+If a tenant named test-user want to receive notification with Wechat Work, it must be create a wechat receiver like this.

```

Comment:
receive notification with Wechat Work => `receive a notification from Wechat Work` might be exact.